.editorconfig

```diff
@@ -10,5 +10,5 @@ trim_trailing_whitespace = true
 [*.md]
 trim_trailing_whitespace = false
 
-[{*.json,.eslintrc,.babelrc}]
-indent_size = 2
\ No newline at end of file
+[*.json]
```

Comment:
All files are now indented at 4 except for json files (in practice just `package.json`, which npm reqrites into 2-space indent)